woocommerce/woocommerce
# Use consistent sanitization on tax classes

### Changes proposed in this Pull Request:

This PR supersedes #30131 to update the bulk edit sanitization to be consistent with the original PR. WooCommerce uses inconsistent sanitization on tax classes where multi-byte characters are inconsistently supported.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open WooCommerce > Settings > Tax
2. Add a multi-byte tax class (eg. `軽減税`)
3. Quick edit a product and change the tax class to this class
4. Edit the product and verify that the tax class was properly saved
5. Bulk edit 2 products and change the tax class to this class
6. Edit the product and verify that the tax class was properly saved
7. Edit a 4th product, set the tax class to this class, and save
8. Go back to the product list
9. Edit the product and verify that the tax class was properly saved



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
